src/group.rs

```diff
@@ -396,7 +392,6 @@ impl<'a> GroupMutIter<'a> {
         }
     }
     /// Inserts the given entry data at the right spot.
-    #[pre("Caller already grew the group by `payload_size + size_of::<ENTRY_HEADER>()`")]
```

Comment:
Please make this a "/// Precondition:" comment or something.

Reply:
Done